src/unix/linux-syscalls.c

```diff
@@ -367,3 +377,25 @@ int uv__statx(int dirfd,
   return errno = ENOSYS, -1;
 #endif
 }
+
+
+int uv__io_uring_register(int fd,
+                          unsigned int opcode,
+                          void *arg,
+                          unsigned int nr_args) {
+  return syscall(__NR_io_uring_register, fd, opcode, arg, nr_args);
+}
+
+
+int uv__io_uring_setup(unsigned int entries, struct uv__io_uring_params *p) {
+  return syscall(__NR_io_uring_setup, entries, p);
+}
+
+
+int uv__io_uring_enter(int fd,
+                       unsigned int to_submit,
+                       unsigned int min_complete,
+                       unsigned int flags, sigset_t *sig) {
+  return syscall(__NR_io_uring_enter, fd, to_submit, min_complete,
+                 flags, sig, _NSIG / 8);
```

Comment:
Using `_NSIG / 8` has been problematic in the past, see 751ac48baaa481202ae321af6f2842c2acf32cdc and 751ac48baaa481202ae321af6f2842c2acf32cdc. The commit logs explain the issue.

I think you might be able to work around it by ensuring you pick up `<asm-generic/signal.h>` first or hard-coding the value:
```c
#ifdef __mips__
#define UV__NSIG 128
#else
#define UV__NSIG 64
#endif
```
(Might not be entirely correct for old-linux-on-sparc but I guess that's okay, we don't support that arch anyway.)